gamemodes/terrortown/gamemode/server/sv_corpse.lua

```diff
@@ -251,32 +273,59 @@ end
 -- @param Player ply
 -- @param Entity corpse
 -- @param boolean is_covert
--- @param boolean is_long_range
+-- @param boolean isLongRange
 -- @return[default=true] boolean
 -- @hook
 -- @register
 -- @realm server
-function GM:TTTCanSearchCorpse(ply, corpse, is_covert, is_long_range)
+function GM:TTTCanSearchCorpse(ply, corpse, is_covert, isLongRange)
 	return true
 end
 
+local function GiveFoundCredits(ply, rag, isLongRange)
+	local corpseNick = CORPSE.GetPlayerNick(rag)
+	local credits = CORPSE.GetCredits(rag, 0)
+
+	if not ply:IsActiveShopper() or ply:GetSubRoleData().preventFindCredits
```

Comment:
`ply:IsActiveShopper()` intended or `ply:Alive() and ply:IsTerror() and ply:IsShopper()`?

Reply:
It has been this way ever since. Why should I use the other one?